src/wallet/wallet2.h

```diff
@@ -486,6 +486,11 @@ namespace tools
     void generate(const std::string& wallet, const epee::wipeable_string& password,
       const cryptonote::account_public_address &account_public_address,
       const crypto::secret_key& viewkey = crypto::secret_key());
+    /*!
+     *
+     */
```

Comment:
Please complete the commentary, or remove it.

Reply:
U+1F612

Reply:
Sorry. I was trying to help.

Reply:
This is right, though. This is pointless AFAICT :P